src/test/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/matrix_build.xml

```diff
@@ -86,9 +86,6 @@
         </hudson.model.Cause_-UpstreamCause>
       </causes>
     </hudson.model.CauseAction>
```

Comment:
I agree that we normally should fix the code not he data.

In this case, the XML is only used by this single test so nothing else is affected by this change. The test case itself is not testing Rebuild plugin at all. Therefore, the test should survive version changes of such plugins. Also, the outer tag '...RebuildAction' still exists in rebuild but the inner tag 'buildurl' has been deprecated long time ago. Its corresponding variable only exists in the class for backwards compatibility reason.